src/main/java/org/broadinstitute/hellbender/engine/GATKTool.java

```diff
@@ -284,10 +296,12 @@ protected void onStartup() {
 
         initializeReads(); // Must be initialized after reference, in case we are dealing with CRAM and a reference is required
 
-        initializeIntervals(); // Must be initialized after reference and reads, since intervals currently require a sequence dictionary from another data source
-
         initializeFeatures();
 
+        initializeIntervals(); // Must be initialized after reference, reads and features, since intervals currently require a sequence dictionary from another data source
+
+        pastInitialize();
```

Comment:
(But see my comment in `VariantWalker` below, where I've suggested deleting this new interface method)